candi/bashible/common-steps/node-group/033_restart_containerd_if_needed.sh.tpl

```diff
@@ -17,6 +17,9 @@ if bb-flag? containerd-need-restart; then
   bb-log-warning "'containerd-need-restart' flag was set, restarting containerd."
   {{- if ne .runType "ImageBuilding" }}
   if out=$(containerd config dump 2>&1); then
+    if [[ -f /etc/systemd/system/containerd.service ]] || [[ -f /usr/lib/systemd/system/containerd.service ]] || [[ -f /lib/systemd/system/containerd.service ]]; then
```

Comment:
Better way move this code above, before `if bb-flag? containerd-need-restart; then`

Reply:
We discussed what to leave like this

Reply:
Or we can check that the `containerd` process is running, turn it off and turn on `containerd-deckhouse` _always_ and not only when the flag is set

Reply:
What do you think about move this code to the bashible first steps ?